### ISSUE TITLE
requirement oddities

### ISSUE THREAD

**Issue by marcverhagen** [clamsproject/app-spacy-wrapper#17]
Using the requirements.txt file in a Docker build crashed the pip install process (it does not crash when I run it in OSX, it just says there are version incompatibilities). So doing all the installs in separate steps because that way it does not crash.

This was an issue a while ago and possibly related issues have popped up, for example, when I do a

```
$ pip install clams-python==0.5.1
$ pip freeze > requirements.txt
```

And then in a fresh environment

```
$ pip install -r requirements.txt
```

This results in an error. Should look at that a bit closer.

**Comment by keighrim** [comment]
This was probably caused by python version error in `0.5.1` images (base images had python 3.10) because some of "capped" dependencies in SDK 0.5.1 had problems with python 10. Thus this issue now should be solved via clams-python 0.5.2 (https://github.com/clamsproject/clams-python/pull/107). The current `develop` branch here is now based on SDK 0.5.2, and i had no issues building/running an image from [the Dockerfile](https://github.com/clamsproject/app-spacy-nlp/blob/bf2cc7d1d37adf33b00747f615b407b5d97da67b/Dockerfile). I believe this issue is ready to be closed, but having one more verification might be a good idea. Leaving open for now.

**Comment by keighrim** [comment]
Never heard about more problems, closing the issue as resolved.
